ee/vellum_ee/workflows/display/nodes/vellum/try_node.py

```diff
@@ -30,9 +30,24 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
         node_display_class = get_node_display_class(BaseNodeVellumDisplay, inner_node)
         node_display = node_display_class(inner_node)
 
-        serialized_node = node_display.serialize(
-            display_context,
-            error_output_id=str(self.error_output_id or uuid4_from_hash(f"{node_display.node_id}|error_output_id")),
-        )
+        serialized_node = node_display.serialize(display_context)
+
+        serialized_node_data = serialized_node.get("data")
+        if isinstance(serialized_node_data, dict):
+            serialized_node_data["error_output_id"] = str(
+                self.error_output_id or uuid4_from_hash(f"{node_display.node_id}|error_output_id")
+            )
```

Comment:
Try nodes annotate their inner nodes. So, I think TryNodeDisplay should annotate its inner display, instead of passing error output id

Reply:
I suppose error_output_id is node specific. definition isn't though - that will be universal. So proposal:
- some notion of context equivalent for error output id (or I'll just add back the kwarg if more than >15 min LoE)
- definition manipulation stays on the outside here

Reply:
agreed!